pkg/schedule/filter/filters.go

```diff
@@ -325,6 +326,8 @@ type StoreStateFilter struct {
 	AllowFastFailover bool
 	// Set true if allows temporary states.
 	AllowTemporaryStates bool
+	// set if operator should not low
+	Level constant.PriorityLevel
```

Comment:
Suggest using a better naming. Or just using a bool. When we init the StoreStateFilter, we can compare it with the low level.

Reply:
yes, it has been renamed as `Operator Level` to set which level the filter used.